java/ql/src/experimental/Security/CWE/CWE-036/OpenStream.ql

```diff
@@ -0,0 +1,56 @@
+/**
+ * @name Calling openStream on URLs created from remote source can lead to file disclosure
+ * @description If openStream is called on a java.net.URL, that was created from a remote source
+ *              an attacker can try to pass absolute URLs starting with file:// or jar:// to access
+ *              local resources in addition to remote ones.
```

Comment:
Our convention is to use the pattern "Syntax X causes behavior Y", which is exactly what you used for the query name. Suggest leaving the more detailed explanation for the `qhelp` file and changing this to:

```suggestion
 * @description Calling openStream on URLs created from remote source 
 * can lead to local file disclosure.
```